raiden/utils/ethereum_clients.py

```diff
@@ -30,19 +42,12 @@ def is_supported_client(client_version: str) -> Tuple[bool, Optional[EthClient],
         ) and our_version <= parse_version(HIGHEST_SUPPORTED_PARITY_VERSION)
         return supported, EthClient.PARITY, str(our_version)
     elif client_version.startswith("Geth"):
-
-        if client_version.startswith("Geth/"):
-            # then this is a geth client version from web3.version.node
-            matches = re.search(r"/v(\d+\.\d+\.\d+)", client_version)
-        else:
-            # result of `geth version`
-            matches = re.search("Version: (.*)-", client_version)
-        if matches is None:
+        our_geth_version = parse_geth_version(client_version)
+        if our_geth_version is None:
```

Comment:
just my OCD again! Since you changed `our_version` to `our_geth_version` here can you also change the name of `our_version` for parity above?

Reply:
Sure. I just changed it because mypy is irritated by one having the type `tuple` and the other being `Optional[tuple]`.